selfdrive/ui/qt/offroad/settings.cc

```diff
@@ -100,7 +100,9 @@ TogglesPanel::TogglesPanel(SettingsWindow *parent) : ListWidget(parent) {
   }
 
   // Toggles with confirmation dialogs
+#ifndef SUNNYPILOT
   toggles["ExperimentalMode"]->setActiveIcon("../assets/img_experimental.svg");
+#endif
```

Comment:
Why this? we didn't do this before

Reply:
We used to comment it out in the derived class, since we do not process any icons in the toggle list, so this is here so the behavior is preserved for stock UI.

Reply:
Synced offline and I see now the diff, great call!